packages/fxa-auth-server/lib/senders/emails/templates/verifySecondary/index.mjml

```diff
@@ -0,0 +1,30 @@
+<%# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I just realized something weird with `verify`, though not directly related to your PR since this is not applicable to `verifySecondary`.

Is it supposed to show the connect another device image instead of the FF logo? 🤨 That's the way it is with the old templates, but it feels off to me.

<img width="1627" alt="image" src="https://user-images.githubusercontent.com/13018240/132390161-548c84fd-4365-420f-bd04-b3d678b5581d.png">


Also, if it is supposed to be this, it looks sort of squished horizontally. Looks like the true dimension of this image is `395px` by `180px`, so if we want it to be `137px` high, the width should be `300px`, not `270px`.

Reply:
Circling back here, I see this is because `sync: true` is in the stories file. In the set of templates I'm working on, I added a story for sync as well as non-sync to show that the devices image shows instead of the FF logo, but I guess that affects all templates since that conditional is in the layout file?

Reply:
Makes sense!

Reply:
For posterity: we're going to create a Story for "layouts" and show the difference there